kmod/src/util.h

```diff
@@ -17,4 +17,15 @@ static inline void down_write_two(struct rw_semaphore *a,
 	down_write_nested(b, SINGLE_DEPTH_NESTING);
 }
 
+/*
+ * When returning shrinker counts from scan_objects, we should steer
+ * clear of the magic SHRINK_STOP and SHRINK_EMPTY values, which are near
+ * ~0UL values. Hence, we cap count to ~0L, which is arbitarily high
+ * enough to avoid it.
+ */
+static inline unsigned long shrinker_min_t_long(unsigned long count)
```

Comment:
Making the input long makes sense as the callers are all longs (or ints).  They onl u64s came from scaling the cached block count to pages.  But if we're not doing that anymore then we can use longs everywhere.  If that's the case, then min here doesn't need to be min_t if both count and LONG_MAX are longs.  And all the callers don't need (u64) casts as they call here (which just gets truncated anyway).